src/explore-education-statistics-admin/src/pages/publication/components/PublicationPublishedReleases.tsx

```diff
@@ -1,139 +1,169 @@
+  MutableRefObject,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import { generatePath, useHistory } from 'react-router';
 
-const pageSize = 5;
-
 interface Props {
   publicationId: string;
-  releases: Release[];
+  pageSize?: number;
+  refetchRef?: MutableRefObject<() => void>;
 }
 
-const PublicationPublishedReleases = ({ publicationId, releases }: Props) => {
+export default function PublicationPublishedReleases({
+  publicationId,
+  pageSize = 5,
+  refetchRef,
+}: Props) {
   const history = useHistory();
 
-  const focusRef = useRef<HTMLTableRowElement>(null);
-
   const [amendReleaseId, setAmendReleaseId] = useState<string>();
-
-  const [focusIndex, setFocusIndex] = useState<number | undefined>(undefined);
+  const [focusReleaseId, setFocusReleaseId] = useState<string>();
 
   const [
     showPublishedStatusGuidance,
     togglePublishedStatusGuidance,
   ] = useToggle(false);
 
-  const [currentPage, setCurrentPage] = useState<number>(1);
+  const {
+    data: releases,
+    hasNextPage,
+    fetchNextPage,
+    isFetchingNextPage,
+    isLoading,
+    isSuccess,
+    refetch,
+  } = useInfiniteQuery(
+    ['publicationPublishedReleases', publicationId],
+    ({ pageParam = 1 }) => {
+      return publicationService.listReleases<ReleaseSummaryWithPermissions>(
+        publicationId,
+        {
+          live: true,
+          page: pageParam,
+          pageSize,
+          permissions: true,
+        },
+      );
+    },
+    {
+      getNextPageParam: lastPage =>
+        lastPage.paging.totalPages > lastPage.paging.page
+          ? lastPage.paging.page + 1
+          : undefined,
+      staleTime: Infinity,
+    },
+  );
+
+  useEffect(() => {
+    if (refetchRef) {
+      // eslint-disable-next-line no-param-reassign
+      refetchRef.current = refetch;
+    }
+  });
 
-  const currentReleases = useMemo(() => {
-    return releases.slice(0, pageSize * currentPage);
-  }, [releases, currentPage]);
+  const lastPage = last(releases?.pages);
 
-  const showMoreNumber =
-    pageSize < releases.length - currentReleases.length
-      ? pageSize
-      : releases.length - currentReleases.length;
+  const allReleases = useMemo(
+    () => releases?.pages.flatMap(page => page.results) ?? [],
+    [releases?.pages],
+  );
 
-  useEffect(() => {
-    if (focusIndex) {
-      focusRef.current?.focus();
+  const showMoreNumber = useMemo(() => {
+    if (!lastPage || !allReleases.length) {
+      return 0;
     }
-  }, [focusIndex]);
+
+    const remainingResults = lastPage.paging.totalResults - allReleases.length;
+
+    return remainingResults < pageSize ? remainingResults : pageSize;
+  }, [allReleases.length, lastPage, pageSize]);
 
   return (
     <>
-      <table
-        className="dfe-hide-empty-cells"
-        data-testid="publication-published-releases"
+      <h3 aria-atomic aria-live="polite">
+        {`Published releases${
+          lastPage ? ` (${lastPage.paging.totalResults})` : ''
+        }`}
```

Comment:
This is meant to show the number of publications currently show and the total e.g. "Published releases (5 of 7)".

Reply:
I spoke to Marv about this last week, as it didn't really make a whole load of sense to me at the time as just displaying the total number of releases seemed sufficient. He wasn't too fussed either way so I just got rid of the lower bound.

Having implemented the pagination properly now, the lower bound kind of makes sense again. Have added it back in!